lib/ansible/modules/network/fortimanager/fmgr_secprof_ssl_ssh.py

```diff
@@ -0,0 +1,1095 @@
+      - Dictionaries must use FortiManager API parameters, not the ansible ones listed below.
+      - If submitted, all other prefixed sub-parameters ARE IGNORED.
+      - This object is MUTUALLY EXCLUSIVE with its options.
+      - We expect that you know what you are doing with these list parameters, and are leveraging the JSON API Guide.
+      - WHEN IN DOUBT, USE THE SUB OPTIONS BELOW INSTEAD TO CREATE OBJECTS WITH MULTIPLE TASKS
+    required: false
+
+  ssl_server_ftps_client_cert_request:
+    description:
+      - Action based on client certificate request failure during the FTPS handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_server_https_client_cert_request:
+    description:
+      - Action based on client certificate request failure during the HTTPS handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_server_imaps_client_cert_request:
+    description:
+      - Action based on client certificate request failure during the IMAPS handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_server_ip:
+    description:
+      - IPv4 address of the SSL server.
+    required: false
+
+  ssl_server_pop3s_client_cert_request:
+    description:
+      - Action based on client certificate request failure during the POP3S handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_server_smtps_client_cert_request:
+    description:
+      - Action based on client certificate request failure during the SMTPS handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+  ssl_server_ssl_other_client_cert_request:
+    description:
+      - Action based on client certificate request failure during an SSL protocol handshake.
+      - choice | bypass | Bypass.
+      - choice | inspect | Inspect.
+      - choice | block | Block.
+    required: false
+    choices: ["bypass", "inspect", "block"]
+
+
+'''
+
+EXAMPLES = '''
+  - name: DELETE Profile
+    fmgr_secprof_ssl_ssh:
+      host: "{{inventory_hostname}}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      name: "Ansible_SSL_SSH_Profile"
+      mode: "delete"
+
+  - name: CREATE Profile
+    fmgr_secprof_ssl_ssh:
+      host: "{{inventory_hostname}}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      name: "Ansible_SSL_SSH_Profile"
```

Comment:
If you are not using variables for particular values, you need not to enclose the values in double quotes, as:
```suggestion
      name: Ansible_SSL_SSH_Profile
```
but, as mentioned above following will remain as it is as its calling variable:
```
host: "{{inventory_hostname}}"
```